Doc/c-api/tuple.rst

```diff
@@ -167,7 +167,13 @@ type.
 
    .. c:member:: const char *name
 
-      Name of the struct sequence type.
+      Pointer to a NUL-terminated string containing the fully qualified name of the
+      type.  For types that are accessible as module globals, the string should be
```

Comment:
I suggest to replace the whole paragraph with just:

"Fully qualified name of the type; null-terminated UTF-8 encoded."

Reply:
Fine with me. I think it would have been enough to put me on the right track when I ran into the issue.